bin/git-theta-filter

```diff
@@ -85,7 +81,8 @@ def smudge(args):
         model_dict[keys] = file_io.load_tracked_file(param_file)
 
     model_dict = utils.unflatten(model_dict)
-    model_checkpoint = checkpoints.PyTorchCheckpoint(model_dict)
+    checkpoint_class = checkpoints.get_checkpoint_class()
```

Comment:
Not something to solve in this PR, but I think we need a better way to specify the checkpoint type. The number of git commands that run smudge/clean will make it hard to remember all the places that need to set this env variable.

Is there a way to configure the `.gitattributes` that track a particular model checkpoint to have it pass an extra argument to clean/filter that we use to set the checkpoint type?

Reply:
Yeah agreed. As far as I can tell it isn't possible to specify extra command line args to clean/smudge for that purpose. Will have to look into this a bit more.